byterun/win32.c

```diff
@@ -819,6 +819,13 @@ CAMLexport int win_wide_char_to_multi_byte(const wchar_t *s, int slen, char *out
   else
     retcode = WideCharToMultiByte(CP_THREAD_ACP, 0, s, slen, out, outlen, NULL, NULL);
 
+  return retcode;
+}
+
+CAMLexport int win_wide_char_to_multi_byte(const wchar_t *s, int slen, char *out, int outlen)
+{
+  int retcode = win_wide_char_to_multi_byte_noexc(s, slen, out, outlen);
+
   if (retcode == 0)
```

Comment:
This code is almost certainly now eliminated, but this should be `if (retcode == 0 && slen != 0)`

Reply:
Indeed, this was the cause for a previous AppVeyor failure, but is now gone.

Reply:
(that was causing your previous AppVeyor failure, since the building of dynlink includes `-ccopt ""` so you were hitting the case of an empty string being passed to `win_wide_char_to_multi_byte`)